django_keycloak/config.py

```diff
@@ -48,22 +48,18 @@ def __post_init__(self) -> None:
         self.KEYCLOAK_URL = f"{URL}{self.BASE_PATH}"
 
 
-__desired_variables = Settings.__annotations__.keys()
-__defined_variables = getattr(
+# Get keycloak configs from django
+__configs = getattr(
```

Comment:
Can this also be written as `__configs = django_settings.KEYCLOAK_CONFIG` ? It'll through an error if not defined that should be clear enough for the user

Reply:
Sure